.github/workflows/docs-and-linting.yml

```diff
@@ -0,0 +1,49 @@
+name: Render and Lint Documentation
+
+on:
+  pull_request:
+    branches_ignore: []
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.10', '1.11', '1.12']
+
+    name: Documentation and Linting
+    steps:
+
+      - uses: actions/checkout@v2
+        with:
+          path: go/src/github.com/opencontainers/image-spec
+
+                               # commit for v1 release
+      - uses: actions/setup-go@0caeaed6fd66a828038c2da3c0f662a42862658f
```

Comment:
wait, this isn't for the image-spec v1, but some github action's release?

Reply:
Yes, I am using the full commit because it’s safer than even a tag or release. https://github.community/t/are-github-actions-safe-to-use/17895/2